### PR TITLE
2020.2.1 Asteria Release Notes

### PR DESCRIPTION
**Pull request by kasenvr**
This PR is based on the security PR, that should be merged first. :) https://github.com/kasenvr/vircadia-docs-sphinx/pull/52

### CODE REVIEW

File: docs/source/host/server-setup/linux-server.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ sudo apt-get update && sudo apt-get install ./vircadia-server_2020.2.0-asteria-2
 ### Amazon Linux 2
 
 ```sh
-sudo yum install https://cdn.vircadia.com/dist/domain-server/amazon-linux/athena-server-0.86.0_K2_20200402_76aff7a-1.amzn2.x86_64.rpm
+sudo yum install https://cdn.vircadia.com/dist/domain-server/amazon-linux/vircadia-server-2020.2.0_ASTERIA_20200608_585fb19-1.amzn2.x86_64.rpm
```
Comment (ctrlaltdavid) [inline docs/source/host/server-setup/linux-server.md:19]: This is still 2020.2.0.

File: docs/source/index.rst [hunk 2]
```diff
@@ -22,6 +22,7 @@ You have the power to shape your VR experience in Vircadia.
 * :doc:`HOST <host>` and make immersive experiences to educate, entertain, and connect with your audience.
 * :doc:`CONTRIBUTE <contribute>` to our endeavor by browsing our source code in the repository.
 * :doc:`DEVELOP <developer>` the project and tailor it to your needs, or just to help out.
+* :doc:`SECURITY <security>` information about the project and its components.```
```
Comment (ctrlaltdavid) [inline docs/source/index.rst:25]:
```suggestion
* :doc:`SECURITY <security>` information about the project and its components.

```

File: docs/source/release-notes/2020-2-1-Asteria.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Interface
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/projects/2).
+
+#### General
+
+* Fix not being able to delete entities in serverless domains. (#425)
+* Fix visibility of values in Wearables dialog. (#426)
+* Reduce minimum camera inspect distance. (#432)
+* Update VR login overlay to Vircadia. (#437)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-2-1-Asteria.md:14]:
```suggestion
* Update VR login overlay to have Vircadia logo. (#437)
```

File: docs/source/security/crash-reporting.rst [hunk 4]
```diff
@@ -0,0 +1,10 @@
+###############
+Crash Reporting
+###############
+
+.. toctree::
+    :maxdepth: 1
+    :titlesonly:
+
+    Crash Reporting Privacy <reporting/crash-reporting>
```
Comment (ctrlaltdavid) [inline docs/source/security/crash-reporting.rst:9]: See comment in PR52 regarding possibly unnecessary TOC level.

File: docs/source/host/server-setup/linux-server.md (hunk 1 above)
Comment (kasenvr) [reply docs/source/host/server-setup/linux-server.md:19]: These changes don't change anything for the server particularly so that's still fine. :)

**Pull request merged**